thrift/src/main/thrift/atoms/interactive.thrift

```diff
@@ -0,0 +1,13 @@
+namespace * contentatom.cta
+namespace java com.gu.contentatom.thrift.atom.interactive
+#@namespace scala com.gu.contentatom.thrift.atom.interactive
+
+struct InteractiveAtom {
+  /* the unique ID will be stored in the `atom` data*/
+  2: required string type
```

Comment:
@clloyd I'm talking about why the numbering starts from 2 sometimes and other times 1.